apis/python/src/tiledb/vector_search/index.py

```diff
@@ -44,7 +53,7 @@ def query(
         nprobe: int = 1,
     ):
         """
-        Open a flat index
+        Query a flat index
```

Comment:
Can we do distributed queries in plain flat indices?

Reply:
We can but we haven't implemented that yet